dotnet/src/SemanticKernel/SemanticFunctions/PromptTemplateConfig.cs

```diff
@@ -137,12 +137,12 @@ public class InputConfig
     public CompletionConfig Completion { get; set; } = new();
 
     /// <summary>
-    /// Default backends to use.
+    /// Default AI services to use.
     /// </summary>
     [JsonPropertyName("default_backends")]
```

Comment:
`default_services`

Reply:
yeah that is a huge breaking change I left out on purpose. @shawncal do you want to do this now or later?

Reply:
done